pvlib/scaling.py

```diff
@@ -0,0 +1,320 @@
+"""
+The ``scaling`` module contains functions for manipulating irradiance
+or other variables to account for temporal or spatial characteristics.
+"""
+
+import numpy as np
+import pandas as pd
+
+
+def wvm(cs_series, latitude, longitude, cloud_speed,
+        method, capacity=None, density=41, dt=None):
+    """
+    Compute spatial aggregation time series smoothing on clear sky index based
+    on the Wavelet Variability model of Lave et al [1-2]. Implementation is
+    basically a port of the Matlab version of the code [3].
+
+    Parameters
+    ----------
+    cs_series : numeric or pandas.Series
+        Clear Sky Index time series that will be smoothed.
+
+    latitude : numeric
+        Latitudes making up the plant in degrees. Type depends on method used.
+        'discrete' uses a list of lat/lon points representing discrete sites.
+        'square' uses a single valued lat/lon pair for the plant center.
+        'polygon' uses a list of lat/lon points representing the corners of
+        a polygon making up the site.
+
+    longitude : numeric
+        Longitudes making up the plant in degrees. Type depends on method used.
+        'discrete' uses a list of lat/lon points representing discrete sites.
+        'square' uses a single valued lat/lon pair for the plant center.
+        'polygon' uses a list of lat/lon points representing the corners of
+        a polygon making up the site.
+
+    cloud_speed : numeric
+        Speed of cloud movement in meters per second
+
+    method : string
+        The type of plant distribution to model.
+        Options are ``'discrete', 'square', 'polygon'``.
+
+    capacity : numeric
+        The plant capacity in MW. Must be specified for 'square' method,
+        ignored otherwise.
+
+    density : numeric, default 41
+        The density of installed PV in W/m^2. Must be specified for 'square'
+        method, ignored otherwise. Default of 41 W/m^2 is 1MW per 6 acres.
+
+    dt : numeric
```

Comment:
`float, default None`  Need to mention units (seconds) and add `[s]` at the end of the description.

Let's move `dt` up in the argument list ahead of `capacity` and `density`